plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/active-filters/frontend.ts

```diff
@@ -16,6 +16,10 @@ type ActiveFiltersContext = {
 	item: ActiveFilterItem;
 };
 
+// Stores are locked to prevent 3PD usage until the API is stable.
```

Comment:
This message is not needed in the stores of the blocks that, in reality, will probably never be public.

Reply:
I've removed them in this commit: https://github.com/woocommerce/woocommerce/pull/57232/commits/06a2f45a5e393f5ad224648bf76b0717ce825a71